experimental/packages/api-logs/src/types/AnyValue.ts

```diff
@@ -14,16 +14,29 @@
  * limitations under the License.
  */
 
-import { AttributeValue } from '@opentelemetry/api';
+export type AnyValueScalar = string | number | boolean;
+
+export type AnyValueArray = Array<AnyValue>;
 
 /**
  * AnyValueMap is a map from string to AnyValue (attribute value or a nested map)
  */
 export interface AnyValueMap {
-  [attributeKey: string]: AnyValue | undefined;
```

Comment:
reviewer note: since now `AnyValue` can be `null` or `undefined`, it is not needed to state it here for map specifically